tests/conftest.py

```diff
@@ -15,6 +15,9 @@ def app():
 
 @pytest.fixture
 def client(app):
+    app.config['DEBUG'] = True
+    app.config['TESTING'] = True
```

Comment:
what do those variables do?

Reply:
https://flask.palletsprojects.com/en/1.1.x/config/#DEBUG - probably doesn't actually do anything in this context, and I reckon we can remove this
https://flask.palletsprojects.com/en/1.1.x/config/#TESTING - disables error handler (so if theres an exception you see the stack trace rather than just see the function returning a 500 error page)

Reply:
cool, thanks 👍